test/spec/config/config.spec.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import { assert } from 'chai';
+import { Environment } from '../../../src/modules/config/types';
+import ConfigService from '../../../src/modules/config/config-service';
+
+describe('ConfigService', () => {
```

Comment:
consider adding spec for all environments please.

Reply:
It's very hard to change the config at runtime and make the app throw an `EnvironmentNotSupportedError` error.